### DISCUSSION TITLE
Square(ish) logo?

### DISCUSSION THREAD

**Discussion by al42and** [AdaptiveCpp/AdaptiveCpp#1200] [General]
Current logo is too wide and looks bad when squeezed into a 1:1 proportion, which is how logos and avatars are often treated on social platforms (GitHub, Discord):

![Screenshot_20231020_155422](https://github.com/AdaptiveCpp/AdaptiveCpp/assets/933873/10c04927-276c-41af-9ffa-c0b99cc55815)

Having a version of the logo with just "ACpp" (and a whoosh below, of course) might be useful for such cases.

**Comment by illuhad** [comment]
Since we already use ACPP as an abbreviation in a lot of places (like compiler flags etc) my concern would be that then there might be confusion over what the project is actually called if such an abbreviation is also used in the logo.

Perhaps we could also try to split the text across two lines?
